app/views/invites/new.html.erb

```diff
@@ -18,6 +18,8 @@
         <% end %>
       </ul>
       <br/><br/>
+
+      <%= f.hidden_field(:event_id, value: @event.id) %>
```

Comment:
Do you need this field? even_id should be in url `/events/:event_id/invites/new`

Reply:
The event_id is available in the url of the 'new' page, but does not seem to be available to the 'create' action when the form is submitted. The hidden field was the way we found to pass it to the create action.

Reply:
You can make it available:)

routes:

```
resources :events do
    resources :invites, only: [:new, :create]
  end

  resources :invites, only: [:update]
```

invites_controller:

```
 @event = Event.find(params[:event_id])
```

form:

```
 <%= form_for @invite, url: event_invites_path(@event) do |f| %>
```

Reply:
Thank you! Took me a few minutes to figure out that I had to change the tests, too, but it works.